src/main/kotlin/tanvd/grazi/grammar/Typo.kt

```diff
@@ -30,8 +29,9 @@ data class Typo(val location: Location, val info: Info, val fixes: List<String>
 
     constructor(match: RuleMatch, lang: Lang, offset: Int = 0) : this(
             Location(match.toIntRange().withOffset(offset)),
-            Info(lang, match.rule, match, match.typoCategory), match.suggestedReplacements)
-
+            Info(lang, match.rule, match, match.typoCategory),
+            match.suggestedReplacements.map { LangToolFixes.fixSuggestion(match.rule, it) }
```

Comment:
What if I will use main constructor? Move replace of suggestedReplacements to `init`

Reply:
For LangTool intended only this constructor, so added a comment